.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,67 +1,44 @@
 ---
-name: Bug report
+name: Library-specific Bug Report
 about: Report a problem with this library that needs to be fixed.
 title: ''
 labels: bug, triage
 assignees: ''
 
 ---
-<!--
-**PLEASE READ**
-==============================================================================
-Before posting, make sure you've carefully reviewed the documentation for this
-library:
-
-https://developers.google.com/google-ads/api/docs/client-libs/python
-
-This library is only used for the Google Ads API. For the AdWords API client
-library report your issue here:
-
-https://github.com/googleads/googleads-python-lib/issues
-
-Only use this page to report problems with this library, which is designed for
-the Python language.
+<!-- PLEASE READ
+This form is for bug reports specific to this library.
 
-If you are looking for help with a different language, see:
+For troubleshooting tips, see: https://developers.google.com/google-ads/api/docs/best-practices/troubleshooting
 
-https://developers.google.com/google-ads/api/docs/client-libs
+For issues that are related to the Google Ads API itself and not specific to this library, please reach out to one of our support channels: https://developers.google.com/google-ads/api/support.
 
-If you are unsure whether your issue is related to this library or the Google
-Ads API itself, please first reach out to one of the below resources before
-filing an issue here:
-
-* Support Forum: https://groups.google.com/forum/#!forum/adwords-api
-* Email the Support Team: googleadsapi-support@google.com
-==============================================================================
+Please check existing issues to see if your bug has already been reported before
+creating a new issue. We encourage you to add any relevant information to the existing issue if one already exists.
 -->
 
 **Describe the bug:**
-<!-- Insert a clear and concise description of what the bug is. -->
 
 **Steps to Reproduce:**
-<!-- Insert specific steps to reproduce the behavior. -->
 
 **Expected behavior:**
-<!-- Write a clear and concise description of what you expected to happen. -->
-
-**Screenshots:**
-<!-- If applicable, attach screenshots to help explain your problem. -->
 
-**Library version and API version:**
+**Client library version and API version:**
 <!--
  - Which Google Ads API version are you using [e.g. v6]
  - Paste the list of dependencies you're using (i.e. `pip freeze`)
```

Comment:
pip freeze should go after the client library and API version

Reply:
what do you mean? Like this?
```
**Client library version and API version:**
Client library version:
Google Ads API version:
<!-- Paste the list of dependencies you're using (i.e. `pip freeze`).] -->
```